src/game_picture.h

```diff
@@ -73,31 +73,35 @@ class Game_Picture {
 	void Move(const MoveParams& params);
 	void Erase(bool force_erase);
 
+	// FIXME: Use C++20 span
+	static void Update(std::vector<Game_Picture>& pictures);
+	// FIXME: Use C++20 span
+	static void UpdateSprite(std::vector<Game_Picture>& pictures);
```

Comment:
imo this adds an incentive to add an incentive to add this "FIXME: span" comment everywhere where std::vector is passed by reference and will keep it there for at least 5 years.